nucypher/network/retrieval.py

```diff
@@ -158,9 +157,20 @@ def capsules(self) -> List[Capsule]:
         return [rk.capsule for rk in self.__retrieval_kits]
 
     @property
-    def lingos(self) -> Conditions:
-        _lingos = [rk.conditions for rk in self.__retrieval_kits]
-        rust_lingos = _serialize_rust_lingos(_lingos)
+    def conditions(self) -> Conditions:
+        _conditions_list = [rk.conditions for rk in self.__retrieval_kits]
+        rust_conditions = self._serialize_rust_conditions(_conditions_list)
+        return rust_conditions
+
+    @staticmethod
+    def _serialize_rust_conditions(conditions_list: List[Conditions]) -> Conditions:
+        lingo_lists = list()
+        for condition in conditions_list:
+            lingo = condition
+            if condition:
```

Comment:
Why would a `condition` be "falsy" here?

Reply:
This is a list of conditions corresponding to the list of retrieval kits, some may have conditions associated with them and some may not. It's really only for reencryption request and NOT for cbd. Technically we will be removing support for pre-tdec at some point.

This is why I localized this functionality to the `RetrievalWorkOrder` instead of keeping it as a utility.